server/src/librelay/relay_server_test.go

```diff
@@ -407,7 +408,7 @@ func assertNoTransactionResent(t *testing.T, relay *RelayServer) {
 }
 
 func TestCreateRelayTransaction(t *testing.T) {
-	request := newRelayTransactionRequest(t, 0, "0x717591514ee3e28f3981d9b1a9a80a4d5b8a2a824ad6d4329a83272e14ecf87b77067fa322eb5142d93bedd488d1a34aea2faef8c288cbf4603e55211f34e5171c")
+	request := newRelayTransactionRequest(t, 0, "0xc2f3ccc4a624ca99cbd7928503f33347a3dcaaf624d8640610c0193e7b3ae868409d84ad4a97c8f01c0bd8f9b65bd81a9efd44b865ad75df13bdfb512d809b911b")
```

Comment:
these opaque strings are still here, with no explanation on how to generate them.

Reply:
There is an explanation on how to generate them printed to console if test fails. again, this go code is not staying with us, hopefully.